src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Main {
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[6];
+        for (int i = 0; i < figures.length; i++) {
```

Comment:
first half or array randomFigure and second half default figure

Reply:
figureSupplier have default Method.

Reply:
You should add check if it is first half of array, or second.
For first half use `figureSupplier.getRandomFigure();`, for second - `figureSupplier.getDefaultFigure();`